modules/mono/glue/GodotSharp/GodotSharp/Core/GD.cs

```diff
@@ -238,9 +238,11 @@ public static Variant.Type TypeToVariantType(Type type)
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal extern static void godot_icall_GD_randomize();
 
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        internal extern static double godot_icall_GD_randf_range(double from, double to);
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        internal extern static double godot_icall_GD_rand_range(double from, double to);
+        internal extern static int godot_icall_GD_randi_range(int from, int to);
```

Comment:
This file is missing changing the actual public method that makes use of this internal call: https://github.com/godotengine/godot/blob/18576e69c6ce12fc96c5c9a6ebe1e5ac8d65fc00/modules/mono/glue/GodotSharp/GodotSharp/Core/GD.cs#L130-L133

Reply:
Why didn't the CI checks catch this?

Reply:
Fixed

Reply:
@aaronfranke The CI doesn't build the C# API yet (which requires generating the bindings as well). Hopefully we can add that in the future.